### ISSUE TITLE
Solution for multi-gpu train error

### ISSUE THREAD

**Issue by Fangyi-Chen** [jwyang/faster-rcnn.pytorch#437]
I believe this code was based on 0.3 torch version.  The multi-gpus results are based on this version.  now the author modified it for torch 0.4.0 by adding ''.item()'' behind variables, and this works pretty well on single gpu.  However there is still a bug hard to fix when you use multi-GPUS with torch 0.4.0 because the higher version torch cannot accept scalar as loss.  I highly suggest you guys use torch 0.3.1 and delete all .item() behind Variables, such that you can train with multi-gpus.

**Comment by cherishfive** [comment]
I tried it according to your method, pytorch0.3.1, and deleted all item(), but he still reports an error. Can you give me some suggestions? Thank you
